#!/usr/bin/python3
'''module: 12-raise_exception_msg'''


def raise_exception_msg(message=""):
    '''raises an exception with user specified message'''
    raise NameError(message)
